sim/team-validator.ts

```diff
@@ -64,7 +64,11 @@ export class TeamValidator {
 			if (setProblems) {
 				problems = problems.concat(setProblems);
 			}
-			if (removeNicknames) set.name = dex.getTemplate(set.species).baseSpecies;
+			if (removeNicknames) {
+				if (format.name !== '[Gen 7] Cross Evolution' || !dex.getTemplate(set.name).exists) {
```

Comment:
This would allow any name that matches a template.

You might want to override with the exact template name, to prevent stuff like "pikachu 𝐅𝐔𝐂𝐊" from passing this check.